hera_sim/visibilities/vis_cpu.py

```diff
@@ -2,7 +2,7 @@
 from builtins import range
 import numpy as np
 from scipy.interpolate import RectBivariateSpline
-import healpy
+from astropy_healpix import healpy
```

Comment:
I can't comment on the lines below that use this, but the healpy compatible interface is not the recommended interface.